packages/config/src/layer2s/zksyncera.ts

```diff
@@ -60,13 +60,17 @@ export const zksyncera: Layer2 = {
         sinceTimestamp: new UnixTime(1676268575),
         tokens: ['ETH'],
         description: 'Main rollup contract, additionally serving as an escrow.',
+        upgradableBy: ['zkSync Era Multisig'],
```

Comment:
what about having a `const upgrades` on top of the file, similar to https://github.com/l2beat/l2beat/blob/master/packages/config/src/layer2s/zksynclite.ts#L36?